CHANGELOG.md

```diff
@@ -3,6 +3,50 @@ All notable changes to the .NET Upgrade Assistant will be documented in this fil
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## Unreleased
+### Added
+- Include try-convert version in upgrade-assistant output [(#358)](https://github.com/dotnet/upgrade-assistant/pull/358)
+- Add check so ensure file can be loaded. [(#346)](https://github.com/dotnet/upgrade-assistant/pull/346)
+- Add a command to select a different current project [(#343)](https://github.com/dotnet/upgrade-assistant/pull/343)
+- Add a step to update package dependecies before TFM change [(#342)](https://github.com/dotnet/upgrade-assistant/pull/342)
+- Add IComponentIdentifier and ProjectComponents.AspNetCore [(#340)](https://github.com/dotnet/upgrade-assistant/pull/340)
+- Replace BinaryFormatter.UnsafeDeserialization [(#339)](https://github.com/dotnet/upgrade-assistant/pull/339)
+- Be sure to include log exception information if upgrade steps fail [(#335)](https://github.com/dotnet/upgrade-assistant/pull/335)
+- Add flag to exclude MSBuild runtime libraries [(#327)](https://github.com/dotnet/upgrade-assistant/pull/327)
+- Add package map entries for configuration and EF [(#324)](https://github.com/dotnet/upgrade-assistant/pull/324)
+- Include try-convert as part of build [(#322)](https://github.com/dotnet/upgrade-assistant/pull/322)
```

Comment:
Maybe rephrase as "Include try-convert locally with upgrade-assistant so that it isn't required as an external dependency."

Reply:
This seems like a duplicate entry as well